auth.go

```diff
@@ -566,6 +565,21 @@ func (x *Central) MergeTick() {
 	}
 }
 
+func serializeUserInfo(userId UserId, username string) []byte {
```

Comment:
This function needs a better name, such as `userInfoToAuditTrailJSON()`
Also, why return `[]byte`, if you always cast it to a `string`? Rather just return `string`.
Also, why have two parameters to the function, instead of just taking an `*AuthUser` object?